google-cloud-error_reporting/lib/google/cloud/error_reporting/project.rb

```diff
@@ -0,0 +1,220 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google/cloud/errors"
+require "google/cloud/env"
+require "google/cloud/error_reporting/service"
+require "google/cloud/error_reporting/credentials"
+require "google/cloud/error_reporting/error_event"
+
+module Google
+  module Cloud
+    module ErrorReporting
+      ##
+      # Project
+      #
+      # Projects are top-level containers in Google Cloud Platform. They store
+      # information about billing and authorized users, and they control access
+      # to Stackdriver ErrorReporting. Each project has a friendly name and a
+      # unique ID. Projects can be created only in the [Google Developers
+      # Console](https://console.developers.google.com).
+      #
+      # @example
+      #   require "google/cloud/error_reporting"
+      #
+      #   error_reporting = Google::Cloud::ErrorReporting.new
+      #   error_event = error_reporting.error_event "Error with Backtrace",
+      #                                             timestamp: Time.now
+      #                                             service_name: "my_app_name"
+      #   error_reporting.report error_event
+      #
+      # @see Google::Cloud::ErrorReporting.new
```

Comment:
I don't think YARD's `@see` tag can be used this way, at least not in our gh-pages site. Probably better to just write:

```
See {Google::Cloud::ErrorReporting.new}.
```